shell/tinytest.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+# This is shell script could repeat the tests,
+# generate log file and delete raft tmp file
+# usage:
+# modify following args
+#   count: repeat times
+#   test_test: makefile's test, you can modify makefile to add subtest
+# In the root of this project, run `./shell/tinytest.sh`
+# If some error occur, try `chmod a+x ./shell/tinytest.sh`
+# And also, you can run `make cleanlog` to delete all logs
+
+count=1
+test_test=project2a
+
+
+deleteTmp() {
+    # echo "deleting tinykv rubish"
+    find /tmp/ -name "*raft*" | xargs rm -rf -v
```

Comment:
this should be cleaned automatically, so you'd better change the test logic to clean it instead of using a script.

Reply:
ok, I refactor it